dias/analyzers/dataset_analyzer.py

```diff
@@ -130,19 +136,19 @@ def run(self):
                 )
 
             # Loop over acquisitions
+            flag_tend = 0
             for flag_acq in flag_acqs.keys():
```

Comment:
I think this can be simplified to `for flag_acq, all_flag_files = flag_acqs.items():`